mkdocs.yml

```diff
@@ -218,61 +218,81 @@ nav:
-      #     - 'Releases': variants/protectli_vp66xx/releases.md
-      #     - 'Building manual': variants/protectli_vp66xx/building-manual.md
-      #     - 'Initial deployment': variants/protectli_vp66xx/initial-deployment.md
-      #     - 'Firmware update': variants/protectli_vp66xx/firmware-update.md
-      #     - 'Recovery': variants/protectli_vp66xx/recovery.md
-      #     - 'Hardware Configuration Matrix': variants/protectli_vp66xx/hardware-matrix.md
-      #     - 'Test matrix': variants/protectli_vp66xx/test-matrix.md
-      # - 'Protectli V1210/V1410/V1610':
-      #     - 'Overview': variants/protectli_v1x10/overview.md
-      #     - 'Releases': variants/protectli_v1x10/releases.md
-      #     - 'Building manual': variants/protectli_v1x10/building-manual.md
-      #     - 'Initial deployment': variants/protectli_v1x10/initial-deployment.md
-      #     - 'Firmware update': variants/protectli_v1x10/firmware-update.md
-      #     - 'Recovery': variants/protectli_v1x10/recovery.md
-      #     - 'Hardware Configuration Matrix': variants/protectli_v1x10/hardware-matrix.md
-      #     - 'Test matrix': variants/protectli_v1x10/test-matrix.md
+      - 'Protectli':
+          - 'Overview': unified/protectli/overview.md
+          - 'Releases': unified/protectli/releases.md
+          - 'Building manual': unified/protectli/building-manual.md
+          - 'Initial deployment': unified/protectli/initial-deployment.md
+          - 'Firmware update': unified/protectli/firmware-update.md
+          - 'Recovery': unified/protectli/recovery.md
+          - 'Hardware Configuration Matrix': unified/protectli/hardware-matrix.md
+          - 'Test matrix': unified/protectli/test-matrix.md
+          - 'Openness score': unified/protectli/openness-score.md
+          - 'Protectli FW6':
+              # - 'Overview': variants/protectli_fw6/overview.md
+              - 'Releases': variants/protectli_fw6/releases.md
+              - 'Building manual': variants/protectli_fw6/building-manual.md
+              - 'Installation manual': variants/protectli_fw6/installation-manual.md
+              - 'Hardware Configuration Matrix': variants/protectli_fw6/hardware-matrix.md
+              - 'Test matrix': variants/protectli_fw6/test-matrix.md
+          - 'Protectli V1x11':
+              # - 'Overview': variants/protectli_v1x1x/overview.md
+              - 'Releases': variants/protectli_v1x1x/releases.md
+              - 'Building manual': variants/protectli_v1x1x/building-manual.md
+              - 'Initial deployment': variants/protectli_v1x1x/initial-deployment.md
+              - 'Firmware update': variants/protectli_v1x1x/firmware-update.md
+              - 'Recovery': variants/protectli_v1x1x/recovery.md
+              - 'Hardware Configuration Matrix': variants/protectli_v1x1x/hardware-matrix.md
+              - 'Test matrix': variants/protectli_v1x1x/test-matrix.md
+              - 'Openness score': variants/protectli_v1x1x/openness_score.md
```

Comment:
this link 404s for me in local preview

Reply:
@mkopec The sections other than `Overview` in `Protectli` unified section should 404, because I have not created them yet. I would like to ask if I should just do it like with the `Overview` and copy the sections from all the boards into one file?

Reply:
I think the other sheets should probably stay separate. At least the releases, openness score and hardware configuration pages should be separate I believe